WordPress/src/main/java/org/wordpress/android/modules/AppComponent.java

```diff
@@ -482,6 +483,8 @@
 
     void inject(DomainSuggestionsFragment object);
 
+    void inject(UploadWorker object);
```

Comment:
ℹ️ This can be removed too.

Reply:
Removed in 3b8c99709c344010d361cdefc47a8cd7321e8c8e